tools/nni_cmd/ssh_utils.py

```diff
@@ -57,3 +57,17 @@ def create_ssh_sftp_client(host_ip, port, username, password):
         return sftp
     except Exception as exception:
         print_error('Create ssh client error %s\n' % exception)
+
+def remote_remove_directory(sftp, directory):
```

Comment:
remove_remote_directory is better. verb+noun is right pattern for function names.

Reply:
Fixed.